### PR TITLE
add xnautils

### PR DESCRIPTION
**Pull request by mthang**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mthang** [comment]
@conda-forge-admin can you help to review this recipe.

File: recipes/xnatutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - hatch-vcs
+  run:
+    - python >=3.4
+    - xnat >=0.6
+    - progressbar2 >=3.16.0
+    - future >=0.16
+
+test:
+  commands:
+    - xnat-ls --version
```
Comment (xhochy) [inline recipes/xnatutils/meta.yaml:31]: This needs to be defined as an entry point.

File: recipes/xnatutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - hatchling
+    - hatch-vcs
+  run:
+    - python >=3.4
+    - xnat >=0.6
+    - progressbar2 >=3.16.0
+    - future >=0.16
+
+test:
+  commands:
```
Comment (xhochy) [inline recipes/xnatutils/meta.yaml:30]:
```suggestion
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Comment by mthang** [comment]
@xhochy the conda package failed to be built when using pip checks, so I replaced it with pip list instead. I found some conda recipes using pip list instead of pip check. There are few error messages I experienced when using pip check (see below) and I am not the developer of xnat. Any suggestion?

xnat 0.6.2 has requirement importlib-metadata~=6.8, but you have importlib-metadata 8.5.0.
xnat 0.6.2 has requirement pydicom~=2.3, but you have pydicom 3.0.1.
xnat 0.6.2 has requirement python-dateutil~=2.8.2, but you have python-dateutil 2.9.0.

**Comment by pauldmccarthy** [comment]
@mthang @xhochy I've [updated the xnat feedstock](https://github.com/conda-forge/xnat-feedstock/pull/40), so you should be able to proceed with this PR (allowing ~30 minutes for the new builds to be published).

**Comment by mthang** [comment]
@pauldmccarthy Thank you for the quick action ! @xhochy thank you for your help ! I will to proceed with the new PR again.

**Comment by mthang** [comment]
@xhochy can you please review this recipe when you have time.

**Review by xhochy — APPROVED** [review]

**Comment by mthang** [comment]
@conda-forge-admin updated the meta.yaml according to the recommendation.

**Review by xhochy — APPROVED** [review]

**Pull request merged**